src/main/java/net/sf/jsqlparser/expression/JsonTable.java

```diff
@@ -0,0 +1,200 @@
+package net.sf.jsqlparser.expression;
+
+import net.sf.jsqlparser.parser.ASTNodeAccessImpl;
+import net.sf.jsqlparser.statement.select.AbstractFromitem;
+import net.sf.jsqlparser.statement.select.FromItem;
+import net.sf.jsqlparser.statement.select.FromItemVisitor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class JsonTable extends AbstractFromitem implements FromItem {
```

Comment:
is `expression` really the best package? I don't mind but I would have rather packed this close to normal `Table` or `TableFunction`. No show stopper though.